compute/k8s-services/dependencies.tf

```diff
@@ -251,3 +251,13 @@ locals {
 locals {
   fluxcd_apps_repo_url = "${var.fluxcd_apps_git_provider_url}${var.fluxcd_apps_repo_owner}/${var.fluxcd_apps_repo_name}"
 }
+
+# --------------------------------------------------
+# Inactivity based clean up for sandboxes
+# --------------------------------------------------
+
+locals {
```

Comment:
Instead of intriducting a new variable in services, I think we should just use the existing datasource for determining if it is a sandbox or not, 

Can be rewritten to:

```
locals {
  enable_inactivity_cleanup = (
    var.enable_inactivity_cleanup && data.terraform_remote_state.cluster.outputs.eks_is_sandbox ? true : false
  )
}
```